transform.test.ts

```diff
@@ -381,6 +397,40 @@ const testCases: TestCase[] = [
     }
     `,
   },
+  {
+    input: `
+      import React from 'react';
+      import { observer } from "mobx-react-lite";
+
+      const MyComponent: React.FC = observer(() => {
+        return <span>foo</span>
+      })`,
+    output: `
+      import React from 'react';
+      import { observer } from "mobx-react-lite";
+
+      const MyComponent = observer(() => {
+        return <span>foo</span>
+      })`,
+  },
+  {
+    input: `
+      import React from 'react';
+      import { observer } from "mobx-react-lite";
+
+      type Props = { id: number };
+      const MyComponent: React.FC<Props> = observer((props) => {
+        return <span>{props.id}</span>
+      })`,
+    output: `
+      import React from 'react';
+      import { observer } from "mobx-react-lite";
+
+      type Props = { id: number };
+      const MyComponent = observer((props: Props) => {
+        return <span>{props.id}</span>
+      })`,
+  },
```

Comment:
Can we also test this scenario in combination with `React.SFC` and `React.FunctionComponent` ?